lib/task/jobs/jobWorkerTask.class.php

```diff
@@ -67,6 +68,39 @@ public function getJobsCompleted()
         return $this->jobsCompleted;
     }
 
+    public function getMemoryUsageString()
+    {
+        return sprintf(
+            "Worker memory usage (PHP - VmRSS): %.2fkb - %dkb\n",
+            $this->getPhpReportedMemoryUsage(),
```

Comment:
`getPhpReportedMemoryUsage()` returns a formatted string, but `sprintf` is expecting a float.

Reply:
Fixed.